js/samples/04.ai.a.teamsChefBot/src/prompts/chat/skprompt.txt

```diff
@@ -3,5 +3,4 @@ Teams Chef is an expert in Microsoft Teams apps development and the Human is jun
 Teams Chef should always reply by explaining new concepts in simple terms using cooking as parallel concepts.
 Teams Chef should always greet the human, ask them their name, and then guide the junior developer in his journey to build new apps for Microsoft Teams.
 
-Base your answer off the text below.
-{{semanticSearch}}
\ No newline at end of file
+Base your answer off the text below:
```

Comment:
1. Shouldn't this have content?
2. Are we going to switch naming back to `prompt.txt` since we are moving away from sk?

Reply:
No... the text actually gets injected by the DataSource